docs/pages/getting-started/5-authentication.rst

```diff
@@ -1,16 +1,174 @@
+      "language": "NL",
+      "version": "v3",
+      "legalEntity": "did:nuts:90348275fjasihnva4857qp39hn",
+      "validFrom": "2006-01-02T15:04:05+02:00",
+      "validDuration": "2h"
+    }
+
+The ``type`` must be one of the valid Nuts contract types, currently only ``BehandelaarLogin`` for Dutch and ``PractitionerLogin`` for English are supported.
+The ``language``` selects the correct language, ``NL`` for Dutch and ``EN`` for english. The ``version`` must be ``v3``.
+The ``legalEntity`` must refer to the DID of the current organization. The user either selects an organization to login for, or is already logged in.
+The organization must have a DID as described in :ref:`Getting Started on customer integration <connecting-crm>`.
+``validFrom`` is a RFC3339 compliant time string. ``validDuration`` describes how long the contract is valid for. Time unit strings are used like ``1h`` or ``60m``, the valid time units are: "ns", "us" (or "µs"), "ms", "s", "m", "h". The local system timezone is used to format the date and time string.
+
+The return value looks like:
+
+.. code-block:: json
+
+    {
+      "type": "PractitionerLogin",
+      "language": "EN",
+      "version": "v3",
+      "message": "EN:PractitionerLogin:v3 I hereby declare to act on behalf of CareBears located in CareTown. This declaration is valid from Monday, 2 January 2006 15:04:05 until Monday, 2 January 2006 17:04:05."
+    }
+
+The ``message`` from this result is used by the specific means.
+
+Signing a contract with the employee identity means
+***************************************************
+
+This getting started manual shows how to successfully use employee identity authentication to sign a contract.
+
 Basic requirements
-******************
+==================
+
+To use the employee identity as a means for signing a contract, the following is required:
+
+- the user interacts with the XIS/ECD via a recent browser capable of running javascript.
+- the vendor has a Nuts node running.
+- the Nuts node contains a DID for the organization of the user.
+
+Exposed endpoints
+=================
+
+The employee identity means uses endpoints under ``{{publicurl}}/public/auth/v1/means/employeeid/{sessionID}``.
+
+Starting a session
+==================
+
+Start a session by calling the following API on the Nuts node:
+
+.. code-block::
+
+    POST /internal/auth/v1/signature/session
+
+The body for this call looks like:
+
+.. code-block:: json
+
+    {
+        "means": "EmployeeID",
+        "payload": "<message>"
+		"params": {
+			"employer": "did:nuts:90348275fjasihnva4857qp39hn",
+			"employee": {
+				"identifier": "user@example.com",
+				"initials": "T",
+				"familyName": "Tester",
+				"roleName": "VP"
+			}
+		}
+    }
+
+Where ``message`` is the result from the contract call discussed earlier.
+All fields are mandatory except ``roleName``. The ``identifier`` field must include a unique identifier for the user, for example an email address or employee number.
```

Comment:
currently roleName is required:

```
invalid session params: missing employee roleName
```

Reply:
master branch or PR? Spec PR also has it listed as optional

Reply:
@stevenvegt any good reason you implemented it being required, or just by accident?

Reply:
C'est un accident!